docs/src/pages/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Introduction
+weight: -100
+---
+
+The [HealthCare.gov](https://www.healthcare.gov/) design system is a set of open-source design and front-end development resources for creating usable, accessible (e.g., WCAG/Section 508 compliant), responsive, and consistent websites and web-based solutions.
```

Comment:
Hm, this content reads more like a README, and is pretty detailed. I was personally thinking something more concise and public facing, similar to the screenshot Barb shared [here](https://adhoc.slack.com/archives/C645GLLNN/p1605041312428900?thread_ts=1605034392.421300&cid=C645GLLNN). 

What do you think of that? Sorry, I know you already wrote out all this content!

Reply:
Maybe we could ask designers or Kristen on HC.gov to help with this content?

Reply:
I can get feedback from some more designers